app/src/main/java/org/mifos/selfserviceapp/ui/fragments/LoanAccountSummaryFragment.java

```diff
@@ -0,0 +1,151 @@
+package org.mifos.selfserviceapp.ui.fragments;
+
+/*
+~This project is licensed under the open source MPL V2.
+~See https://github.com/openMF/self-service-app/blob/development/LICENSE.md
```

Comment:
Change ```development``` with ```master``` because right now It is not in master branch but soon we will release and will merge in master branch.